SpringProject/src/main/java/com/gradproject2019/users/repository/UserRepository.java

```diff
@@ -54,4 +54,21 @@ void updateUser(@Param("id") Long id,
             "ORDER BY last_name ASC",
             nativeQuery = true)
     List<User> searchByName(@Param("query") String query);
+
+    @Query(value = "SELECT EXISTS(" +
```

Comment:
Would it work with just SELECT 1 and the method returning a boolean as @ed-steele-rdm-els suggested?

Reply:
It would, but the point of using EXISTS is that the the transaction can end as soon as one row matches (i.e. doesn't have to iterate whole filtered table) SELECT 1 returns 1 for each row that matches, on an app this size it makes no difference but I always prefer `EXISTS` - which would return 1/0 which would need to be mapped to a boolean in either case.

I think returning an int is OK for now (depending on the complexity of binding a result set mapper here)

Reply:
Yeah I did try with returning a boolean but It wasn't having it so I decided to go with the int method. I can put it in a ticket as a fix to do after the feature freeze though?